rfc/sandbox-submission.md

```diff
@@ -0,0 +1,56 @@
+This is a working document to outline responses to request for CNCF sandbox submission.
+
+### links
+* [issue](https://github.com/awslabs/cdk8s/issues/280) 
+* [process](https://github.com/cncf/toc/blob/master/process/project_proposals.adoc#sandbox-process)
+* [form](https://docs.google.com/forms/d/1bJhG1MuM981uQXcnBMv4Mj9yfV5_q5Kwk3qhBCLa_5A/viewform?edit_requested=true)
+
+### responses
+1. **Email address**
+
+...
+
+2. **Project Name**
+Cloud Development Kit for Kubernetes (cdk8s)
+
+3. **Project Description**
+The Cloud Development Kit for Kubernetes (cdk8s) is a software development framework for defining Kubernetes applications and reusable abstractions using familiar programming languages and rich object-oriented APIs. cdk8s generates pure Kubernetes YAML - you can use cdk8s to define applications for any Kubernetes cluster running anywhere.
+
+4. **Code repository (URL)**
+https://github.com/awslabs/cdk8s
+
+5. **website (URL)**
+cdk8s.io
+
+6. **project roadmap**
+https://github.com/awslabs/cdk8s/projects/1
```

Comment:
Should we tidy up our roadmap? I'm not sure how maintained it is...Or is this to make sure we "have one"?

Reply:
this is to make sure we have one. We can tidy but that's separate.